scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -124,6 +124,7 @@ export function AuthoringDirective(
             $scope.mediaFieldVersions = {};
             $scope.refreshTrigger = 0;
             $scope.isPreview = false;
+            $scope.isCorrectionDisabled = false;
```

Comment:
It's disabled only when in progress right? It'd be clearer to call it `isCorrectionInProgress`

Reply:
Yes, you are right.
> It'd be clearer to call it isCorrectionInProgress

Firstly, I thought of the same name. Let me just rename it.